### PR TITLE
feat(web): add icon to action / reaction

### LINKED ISSUE #125: Add icon of service next to action / reaction in area popup
What we currently have:

image

Add little icons next to each reaction and action in area popup showing it's corresponding services.

(Wait until area parameters are done)

### PR DESCRIPTION
**Pull request by Miou-zora**
Linked to:
- close #125

### CODE REVIEW

**Review by Pablodeibar — APPROVED** [review]
:monkey:

**Review by Benjicatch — APPROVED** [review]

**Review by AlexGuillard — COMMENTED** [review]
isn't it dirty to put entire functions in the useeffect?

File: web/src/Component/Add/add.tsx [hunk 1]
```diff
@@ -193,6 +199,34 @@ function Add() {
   }
 
   useEffect(() => {
+    const handleCallActionList = () => {
+      const storedToken = Cookies.get('token');
+      axios.get(process.env.REACT_APP_SERVER_URL + "/" + storedToken + "/actions")
```
Comment (Benjicatch) [inline web/src/Component/Add/add.tsx:204]: call storedToken in header

File: web/src/Component/Add/add.tsx [hunk 2]
```diff
@@ -193,6 +199,34 @@ function Add() {
...
+          ]);
+        })
+        .catch(error => {
+          console.error(error);
+        });
+    }
+
+    const handleCallReactionList = () => {
+      const storedToken = Cookies.get('token');
+      axios.get(process.env.REACT_APP_SERVER_URL + "/" + storedToken + "/reactions")
```
Comment (Benjicatch) [inline web/src/Component/Add/add.tsx:218]: call storedToken in header

**Review by Benjicatch — CHANGES_REQUESTED** [review]

File: web/src/Component/Add/add.tsx (hunk 2 above)
Comment (Miou-zora) [reply web/src/Component/Add/add.tsx]: done

File: web/src/Component/Add/add.tsx (hunk 1 above)
Comment (Miou-zora) [reply web/src/Component/Add/add.tsx]: done

**Comment by Miou-zora** [comment]
> isn't it dirty to put entire functions in the useeffect?

I think it's cleaner + it avoid to have some random shit warning + ratio

**Review by Benjicatch — APPROVED** [review]

**Pull request merged**